content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,675 @@
+
+{{< text bash >}}
+$ kubectl get pods -n ambient-demo
+{{< /text >}}
+
+{{< text syntax=plain snip_id=none >}}
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-648cd984f8-7vg8w   1/1     Running   0          31m
+notsleep-bb6696574-2tbzn   1/1     Running   0          31m
+sleep-69cfb4968f-mhccl     1/1     Running   0          31m
+sleep-69cfb4968f-rhhhp     1/1     Running   0          31m
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl get svc httpbin -n ambient-demo
+{{< /text >}}
+
+{{< text syntax=plain snip_id=none >}}
+NAME      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+httpbin   ClusterIP   10.110.145.219   <none>        8000/TCP   28m
+{{< /text >}}
+
+Note that each application pod has just 1 container running in it (the "1/1" indicator) and that `httpbin` is an http service listening on `ClusterIP` service port 8000. We should now be able to `curl` this service from either client pod and confirm it returns the `httpbin` web page as shown below. At this point there is no `TLS` of any form being used.
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -n ambient-demo  -- curl httpbin:8000 -s | grep title -m 1
+{{< /text >}}
+
+{{< text syntax=plain snip_id=none >}}
+    <title>httpbin.org</title>
+{{< /text >}}
+
+### Enabling Ambient for an application
+
+We now enable `ambient` for the application deployed in the prior subsection by simply adding the label `istio.io/dataplane-mode=ambient` to the application's namespace as shown below. A namespace should not be enabled for both sidecar mode and ambient mode at the same time. For now, we focus on using `ambient` mode only. In a following subsection we describe how conflicts are resolved in hybrid scenarios that mix mix sidecar mode and ambient mode within the same mesh.
+
+{{< text bash >}}
+$ kubectl label namespace ambient-demo istio.io/dataplane-mode=ambient
+$ kubectl  get pods -n ambient-demo
+{{< /text >}}
+
+{{< text syntax=plain snip_id=none >}}
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-648cd984f8-7vg8w   1/1     Running   0          78m
+notsleep-bb6696574-2tbzn   1/1     Running   0          77m
+sleep-69cfb4968f-mhccl     1/1     Running   0          78m
+sleep-69cfb4968f-rhhhp     1/1     Running   0          78m
+{{< /text >}}
+
+Further, we see that after this, we still see only 1 container per application pod and the uptime of these pods indicates these were not restarted in order to enable `ambient` mode (unlike `sidecar` mode which does restart application pods when the sidecar proxies are injected). This results in better user experience and operational performance since `ambient` mode can seamlessly be enabled (or disabled) completely transparently as far as the application endpoints are concerned. We can initiate a `curl` request again from one of the client pods to the service and again verify that this works now while in `ambient` mode.
```

Comment:
```suggestion
Note that after `ambient` is enabled for the namespace, every application pod still only has 1 container, and the uptime of these pods indicates these were not restarted in order to enable `ambient` mode (unlike `sidecar` mode which does restart application pods when the sidecar proxies are injected). This results in better user experience and operational performance since `ambient` mode can seamlessly be enabled (or disabled) completely transparently as far as the application pods are concerned. 

Initiate a `curl` request again to from a client pod to to a service endpoint to verify traffic continues to flow while in `ambient` mode:
```

Reply:
made the suggested changes